traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -257,9 +256,9 @@ func snapshotHandler(w http.ResponseWriter, r *http.Request, deprecated bool) {
 
 // SnapshotOldGUIHandler creates the CRConfig JSON and writes it to the snapshot table in the database. The response emulates the old Perl UI function. This should go away when the old Perl UI ceases to exist.
 func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
-	inf, userErr, sysErr, _ := api.NewInfo(r, []string{"cdn"}, nil)
+	inf, userErr, sysErr, code := api.NewInfo(r, []string{"cdn"}, nil)
```

Comment:
NIT: The other handlers label this variable with `errCode` instead of just `code`